bpf/lib/overloadable_xdp.h

```diff
@@ -106,6 +106,7 @@ ctx_set_encap_info(struct xdp_md *ctx __maybe_unused,
 		   __u32 vni __maybe_unused, __u32 *ifindex __maybe_unused)
 {
 	return DROP_INVALID;
+	/* return CTX_ACT_OK; */
```

Comment:
Leftover? Although I see you remove it in a later commit

Reply:
I temporarily added this in the initial refactor patch, to motivate where the `CTX_ACT_OK` will be returned from later.